geant-incubator/Issues/edugain-proxy/description.xml

```diff
@@ -0,0 +1,113 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo GEANT Incubator eduGAIN proxy</en>
+        <nl>Demo GEANT Incubator eduGAIN proxy</nl>
+	</Name>
+	<ShortName>
+        edugain-proxy
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>geant-incubator</IssuerID>
+	<CredentialID>edugain-proxy</CredentialID>
+	<Description>
+		<en>This issuer is part of the Distributed Identity for Research (DI4R) IRMA demo, part of the the GÉANT Trust and Identity Incubator.\nThis issuer is a proxy between Identity Providers in eduGAIN and the DI4R IRMA ecosystem. It allows users to load credetials from their home institution into an IRMA wallet for login to services lateron.</en>
+   	    <nl>Deze issuer is onderdeel van de Distributed Identity for Research (DI4R) IRMA demo, wat onderdeel is van de GÉANT Trust and Identity Incubator.\nDeze issuer is een proxy tussen de Identity provders in eduGAIN en het DI4R IRMA ecosysteem. Middels deze proxy kunnen gebruikers credentials van hun instelling inladen in een IRMA wallet om later bij diensten te gebruiken om in te loggen.</nl>
+	</Description>
+	<ShouldBeSingleton>false</ShouldBeSingleton>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.incubator.geant-incubator.edugain-proxy.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.incubator.geant-incubator.edugain-proxy.html</nl>
+	</IssueURL>
+	<ForegroundColor>#15222E</ForegroundColor>
+	<BackgroundGradientStart>#EBEBEB</BackgroundGradientStart>
+	<BackgroundGradientEnd>#FFFFFF</BackgroundGradientEnd>
+	<IsInCredentialStore>false</IsInCredentialStore>
+	<Category>
+		<en>Education / research</en>
+	</Category>
+	<FAQIntro>
+		<en>Add your credentials from your education or research institution to your IRMA app.</en>
+		<nl>Voeg credentials van je onderwijs of onderzoeksinstelling toe aan je IRMA app.</nl>
+	</FAQIntro>
+	<FAQPurpose>
+		<en>By adding credentials to your your IRMA Wallet, you may gain access to services provided by your research collaboration.</en>
+	</FAQPurpose>
+	<FAQContent>
+		<en>You may add the following data:\n•home organization domain name\n•shared user identifier\n•person name\n•email address\n•affiliation\n•assurance information</en>
+	</FAQContent>
+	<FAQHowto>
+		<en>To add credentials to your wallet please scan the QR code with your IRMA wallet app.</en>
+	</FAQHowto>
+    <Attributes>
+       <!-- organization -->
+       <Attribute id="schacHomeOrganization" optional="true">
+          <Name>
+             <en>Home organization</en>
```

Comment:
Ideally, this string and all other strings having only an `en` translation would also have `nl` translations.

(If you like, you can skip or even remove the XML tags starting with `FAQ`; those are only used when `IsInCredentialStore` is set to `true`.)

Reply:
Agreed. If we bring this towards production service we would probably also add stings for all other languages in the EU, however, for our current demo setup, EN suffices